fw/connection.c

```diff
@@ -139,6 +139,10 @@ tfw_connection_release(TfwConn *conn)
  * ensure that conn->sk remains valid for the life of @conn instance.
  * The socket itself may have been closed, but not deleted. ss_send()
  * makes sure that data is sent only on an active socket.
+ *
+ * NOTE: after `tfw_connection_send` returns, `msg` should not be used!
+ * See `tfw_tls_conn_send` for reference.
+ *
```

Comment:
I'm not sure if this is the best place to comment this - a code observer interested in `tfw_connection_send()` may not see the header comment

Reply:
moved to the function body.